environment/visibility.csv

```diff
@@ -80,6 +80,7 @@ LKTA,"No obstruction above 07 degrees in the northern quadrants, upto 16 degrees
 LOK1,good,2016-11-15T00:00:00Z,2017-11-02T00:00:00Z
 LOOK,"good. though Lookout Mountain peak is to the NE, with a maximum elevation of 20 degrees.",2017-03-28T00:00:00Z,9999-01-01T00:00:00Z
 LRR1,good,2016-11-15T00:00:00Z,2017-05-22T00:00:00Z
+LY2T,Close hills to the north and east, with angles of elvation between 10 and 20 degrees. Distant hills in the south and west.,2024-08-22T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
This file is .csv, so comma is a columns separator. If you want to put comma in your sentence you have to use quotation around the sentence. Otherwise the automatic checks will not pass.

Reply:
Thanks Aleks, I just noticed that myself and have added a fix for that